src/components/refreshdialog/refreshdialog.js

```diff
@@ -71,14 +76,15 @@ function onSubmit(e) {
 
     const mode = dlg.querySelector('#selectMetadataRefreshMode').value === 'scan' ? 'Default' : 'FullRefresh';
     const replaceAllImages = mode === 'FullRefresh' && dlg.querySelector('.chkReplaceImages').checked;
+    const replaceTrickplayImages = mode === 'FullRefresh' && dlg.querySelector('.chkReplaceTrickplayImages').checked;
 
     options.itemIds.forEach(function (itemId) {
         apiClient.refreshItem(itemId, {
-
             Recursive: true,
             ImageRefreshMode: mode,
             MetadataRefreshMode: mode,
             ReplaceAllImages: replaceAllImages,
+            replaceTrickplayImages: replaceTrickplayImages,
```

Comment:
There's no `ReplaceTrickplayImages` in the server. 🤷‍♂️
```suggestion
            RegenerateTrickplay: replaceTrickplayImages,
```